indico/modules/rb/notifications/reservation_occurrences.py

```diff
@@ -72,60 +72,23 @@ def notify_rejection(occurrence):
 
 
 @email_sender
-def notify_upcoming_occurrence(occurrence):
-    if occurrence.start_dt.date() < date.today():
-        raise ValueError("This reservation occurrence started in the past")
-
+def notify_upcoming_occurrences(occurrences):
     to_list = []
-    reservation_user = occurrence.reservation.booked_for_user
+    for occ in occurrences:
+        if occ.start_dt.date() < date.today():
+            raise ValueError("This reservation occurrence started in the past")
+
+    reservation_user = occurrences[0].reservation.booked_for_user
```

Comment:
Not really, I skip to next occurence if we have an empty list and dont call the `notify_upcoming_occurrences`
```python
            if not same_user_occurrences:
                continue
            same_user_occurrences = sorted(same_user_occurrences, key=attrgetter('start_dt'))
            notify_upcoming_occurrences(same_user_occurrences)
```